2-breakfast-query-graphql-exercise/index.js

```diff
@@ -0,0 +1,50 @@
+const { ApolloServer, gql } = require("apollo-server");
+const ConcertTote = require("./lib/concert_tote");
+
+const db = new ConcertTote();
+
+db.update({key: 'foodItems', by: item => {
+  return item.map(food => {
+    // This is a simple ID builder
+    food.id = "abcdefghijklmnopqrstuvwxyz".split('').filter(l => Math.random() > 0.8 ).join('') + '-' + Math.ceil(Math.random()*100)
+    // you made need to do other things to update the food items
+    // prior to querying them.
+    return food;
+  })
+}})
```

Comment:
I **like this** here because it doubles as quick documentation to the KV store API.

However, this re-assigns a dynamic ID to each food item every time you save `index.js`. This ended up being a time-sink for me, because I had to copy/paste the ID for my test order mutation in the GraphQL IDE.

Would it be possible to have these persisted in `food_items.js` instead? And maybe keep this in here, but not mutate anything? I'm also curious your original thoughts about what the applicant might need to change in here (I assume `quantity` was an original part of the exercise, which makes sense).